spec/cc/analyzer/include_paths_builder_spec.rb

```diff
@@ -42,7 +42,7 @@ module CC::Analyzer
       end
 
       it "keeps sibling files and directories in include_paths" do
-        %w[subdir/ trackable.rb].each do |trackable_file_or_directory|
+        %w[subdir trackable.rb].each do |trackable_file_or_directory|
```

Comment:
Trailing slashes for directories is part of the spec. It's meant to make it a little easier for engine implementors to find directories:

https://github.com/codeclimate/spec/blob/master/SPEC.md (under "Which files to analyze")

So they should probably be kept in unless there's a strong enough reason to force all the engines to update.